modules/ROOT/content-nav.adoc

```diff
@@ -211,18 +221,18 @@
 *** xref:tools/neo4j-admin/validate-config.adoc[]
 ** xref:tools/cypher-shell.adoc[]
 
+* xref:tutorial/index.adoc[]
+//** xref:tutorial/local-causal-cluster.adoc[]
+//** xref:tutorial/causal-backup-restore-db.adoc[]
```

Comment:
are these supposed to be commented out? I guess yes cause they seemed to have been in the block before it moved up in the file 🤷